docs/source/dev-guide/plugins/environment_exporters.rst

```diff
@@ -0,0 +1,411 @@
+======================
+Environment Exporters
+======================
+
+The environment exporter plugin hook allows you to create custom export formats for conda environments.
+This feature was introduced to enhance the ``conda export`` command with a plugin-based architecture
+for extending conda's environment export capabilities.
+
+Overview
+========
+
+Environment exporters transform conda environments into various file formats that can be shared,
+stored, or used to recreate environments. The plugin system supports both structured formats
+(like YAML/JSON) and text-based formats (like explicit URLs or requirement specs).
+
+Built-in exporters include:
+
+* **environment-yaml** - YAML format for cross-platform environments
+* **environment-json** - JSON format for programmatic processing
+* **explicit** - CEP 23 compliant explicit URLs format
```

Comment:
```suggestion
* **explicit** - `CEP 23 <https://conda.org/learn/ceps/cep-0023>`_ compliant explicit URLs format
```

Let's link to the CEP because we can 😉. 99% sure that's the right syntax 😅